src/Scene/NodeProcess.js

```diff
@@ -215,23 +199,27 @@ function findAncestorWithValidTextureForLayer(node, layerId) {
     }
 }
 
-function updateNodeImagery(quadtree, node, colorLayers) {
-    var promises = [];
+function updateNodeImagery(quadtree, node, layersConfig, force) {
+    let promises = [];
 
-    for (var i = 0; i < colorLayers.length; i++) {
-        var layer = colorLayers[i];
+    const colorLayers = layersConfig.getColorLayers();
+    for (let i = 0; i < colorLayers.length; i++) {
+        let layer = colorLayers[i];
 
-        if (layer.tileInsideLimit(node, layer)) {
-            var args = {
+        // only request visible downscaled color layers
```

Comment:
see Freeze layer #165 
Freeze option layer allows not to load data and let visible layer